src/main/groovy/com/wooga/gradle/io/BaseObject.groovy

```diff
@@ -0,0 +1,46 @@
+package com.wooga.gradle.io
+
+import com.wooga.gradle.BaseSpec
+import org.gradle.api.DefaultTask
+import org.gradle.api.Project
+import org.gradle.api.file.ProjectLayout
+import org.gradle.api.model.ObjectFactory
+import org.gradle.api.provider.ProviderFactory
+
+/**
+ * An object with access to the factories provided by a {@link org.gradle.api.Project}
+ */
+class BaseObject {
```

Comment:
Why do we have this object? It serves the same purpose (access to gradle factories) as the ones it wraps.

Reply:
Will remove this; it was part of another design for the executable class.